docs/static/releasenotes.asciidoc

```diff
@@ -55,12 +56,79 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-13-0]]
+=== Logstash 8.13.0 Release Notes
+
+[[featured-8-13-0]]
+==== New {ls} plugin: Elastic_integration filter
+
+With the {ls} Elastic_integration filter, you can take advantage of the features in {integrations-docs}[Elastic integrations] without missing out on the advanced processing capabilities offered by {ls}. 
+Together, Elastic integrations and {ls} can help you solve a wide range of use cases, including air gapped environments, data privacy processing, and multiple destinations. 
+
+Your {ls} pipeline would include the elastic_agent input, the elastic_integration filter, and the elasticsearch output, in addition to any other plugins you want to use.
+
+Check out <<ea-integrations>> and the <<plugins-filters-elastic_integration,`Elastics_integration filter`>>  plugin docs for details.
+
+[[notable-8.13.0]]
+==== Enhancements and notable issues fixed
+
+* Flow metrics added `worker_utilization` to pipeline-level https://github.com/elastic/logstash/pull/15912[#15912]
+* Ensure that the order of JVM options from both files and environment variables is respected https://github.com/elastic/logstash/pull/15997[#15997]
+
+[[plugin-changes-8.13.0]]
+==== Changes to bundled plugins
+
+* Packaged `filter-elastic_integration` as a bundled plugin in preparation for GA. https://github.com/elastic/logstash/pull/15769[#15769]
+* Unbundled `logstash-input-imap` plugin https://github.com/elastic/logstash/pull/15985[#15985]
+
+==== Plugins
+
+*Elastic_integration Filter - 0.1.6*
+
+* Using this filter you can process Elastic Integrations in Logstash.
+** Once configured to point to an Elasticsearch cluster, this filter will detect which Elastic Integration processing (if any) should be executed for each event by auto-detecting the event’s data-stream.
+** It will then perform that processing inside Logstash without transmitting the event to Elasticsearch. Events that are successfully handled will be tagged so that any downstream Elasticsearch output in the Logstash pipeline will avoid running the Elastic Integration again in Elasticsearch.
```

Comment:
Good info on how this works.  Rather than removing it, I made minor suggestions on messaging and formatting.  If you accept these suggestions, I'll do one final quick check to be sure that it's formatting as I expect it to.

Reply:
Do you think we can merge this with line 81 ?
* Packaged `filter-elastic_integration` as a bundled plugin in preparation for GA. https://github.com/elastic/logstash/pull/15769[#15769]

Both say a plugin is bundled and available for GA

Reply:
I was trying to reach both kinds of readers--the ones who focus on the core updates and the ones who focus on plugins.  I added the lead in to the integrations filter plugin section as an explanation for why we're talking about plugin capabilities in the plugin change log. 

If you don't think it belongs in the plugin section, I think it should come out.  
It's good info, and I can use it somewhere else outside of release notes.